src/main/java/seedu/eylah/expensesplitter/logic/commands/AddItemCommand.java

```diff
@@ -57,16 +57,19 @@ public AddItemCommand(Item item, ArrayList<Person> persons, Amount amount) {
 
     @Override
     public CommandResult execute(SplitterModel splitterModel) throws CommandException {
+
         requireNonNull(splitterModel);
         if (splitterModel.isReceiptDone()) {
             return new CommandResult(MESSAGE_RECEIPT_DONE);
         } else {
             splitterModel.addEntry(toBeAdded);
             for (Person person : persons) {
+
                 if (!splitterModel.hasPerson(person)) {
                     splitterModel.addPerson(person);
                 } else {
                     splitterModel.addAmount(splitterModel.getPerson(person), amount);
+
                 }
```

Comment:
I think this is a good practice to separate different part of code by an empty line.